docs/source/starters/starters.md

```diff
@@ -0,0 +1,81 @@
+# Kedro starters
+
+A Kedro starter contains code in the form of a [Cookiecutter](https://cookiecutter.readthedocs.io/) template for a Kedro project. Using a starter is like using a pre-defined layout when creating a presentation or document.
+
+You can create your own starters for reuse within a project or team, as described in the [how to create a Kedro starter](../starters/create_a_starter.md) documentation.
+
+## How to use a starter
+
+To create a Kedro project using a starter, apply the `--starter` flag to `kedro new`. For example:
+
+```bash
+kedro new --starter=<path-to-starter>
+```
+
+```{note}
+`path-to-starter` could be a local directory or a VCS repository, as long as [Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/usage.html) supports it.
+```
+
+## Starter aliases
+
+We provide aliases for common starters maintained by the Kedro team so that you don't have to specify the full path. For example, to create a project using the `spaceflights-pandas` starter:
+
+```bash
+kedro new --starter=spaceflights-pandas
+```
+To list all the aliases we support:
+
+```bash
+kedro starter list
+```
+
+## Official Kedro starters
+
+The Kedro team maintains the following starters for a range of Kedro projects:
+
+* [`astro-airflow-iris`](https://github.com/kedro-org/kedro-starters/tree/main/astro-airflow-iris): An example project using the [Iris dataset](https://www.kaggle.com/uciml/iris) with a minimal setup for deploying the pipeline on Airflow with [Astronomer](https://www.astronomer.io/).
```

Comment:
(Minor) Do we really want to have this list spelled out here? We'd need to keep it in sync with the repository